### ISSUE TITLE
Make it explicit that mDNS resolution is usually limited to same local network.

### ISSUE THREAD

**Issue by youennf** [rtcweb-wg/mdns-ice-candidates#133]
Section 4.1 assumes that devices are on the same network.
It might be good to be more explicit (maybe in the introduction) that local candidates are mostly useful when two devices are on the same network, which is where mDNS is actually useful.

**Comment by youennf** [comment]
Taken from Christer review at https://mailarchive.ietf.org/arch/msg/mmusic/vrCGrYzOeqWv3ceUIcLKTNIkOXE/

**Comment by juberti** [comment]
Agree, this seems worth of its own paragraph in the intro, and probably also a brief mention at the start of Section 5.
